Sprintfinity3902/Content/Rooms/Room1.csv

```diff
@@ -4,6 +4,7 @@ LFBK, , , , , , , , , , ,RFBK,
  , , , , , , , , , , , 
  , , , , ,DODO, , , , , , 
  , , , , , , , , , , , 
+ , , ,RPSK, , , , , , , , 
```

Comment:
Before final submission we should change these rooms to the original dungeon? Not that anyone's gonna see, but a thought.